helm/charts/hydra/templates/hpa.yaml

```diff
@@ -14,20 +14,20 @@ spec:
   minReplicas: {{ .Values.deployment.autoscaling.minReplicas }}
   maxReplicas: {{ .Values.deployment.autoscaling.maxReplicas }}
   metrics:
-  {{- if .Values.deployment.autoscaling.cpuAverageValue }}
+  {{- if .Values.deployment.autoscaling.targetCPUUtilizationPercentage }}
```

Comment:
Maybe use a if/else block to allow both?

Reply:
yeah  i can do this. is it ok that  i do it in a separate PR as i want to use the implementations in this PR in our tool?

Reply:
Yes of course

Reply:
Or maybe it would be better in this PR - otherwise we are breaking backwards compatibility with the currently released version

Reply:
done